### ISSUE TITLE
FTC compatibility

### ISSUE THREAD

**Issue by EMH333** [EthanHans3n/FRC-Elo-App#29]
This might be reaching a little far however it might be worth a shot. Might it be possible to also allow two teams per alliance for FTC instead of three? My quick look into the logic reveals some minor changes that would be required to change the match entry to support two teams per alliance instead of three. 

Again, I am not underestimating the work required to implement this but I don't see any competing applications on the FTC side and this would broaden the reach of this application.

**Comment by 1ethanhansen** [comment]
Yeah that would definitely be possible, but is a stretch goal as this was designed originally for FRC
